webapp/src/adapters/solid/FriendManager.ts

```diff
@@ -41,11 +42,11 @@ export default class FriendManager {
      * @return The object User corresponding to the POD of the webID
      * */
     public async getUserData(webID: string): Promise<User> {
-        let webIdDoc = await getSolidDataset(webID, {fetch: this.sessionManager.getSessionFetch()});
-        let friends = getThing(webIdDoc, webID);
+        let webIdDoc = await getSolidDataset(webID + "profile/card", {fetch: this.sessionManager.getSessionFetch()});
```

Comment:
The webID already ends in /profile/card#me. With that code you will end up with a string like https://davidmc.inrupt.net/profile/card#meprofile/card

Reply:
The webID is the basePath sorry, I alredy changed that.